src/Akeneo/Platform/Job/back/Infrastructure/Query/FindJobUsers.php

```diff
@@ -26,25 +27,39 @@ public function __construct(Connection $connection, NotVisibleJobsRegistry $notV
         $this->notVisibleJobsRegistry = $notVisibleJobsRegistry; #TODO RAC-1013
     }
 
-    public function search(int $page): array
+    public function search(FindJobUsersQuery $query): array
     {
+        $page = $query->page;
+        $size = $query->size;
+        $username = $query->username;
+
         $notVisibleJobsCodes = $this->notVisibleJobsRegistry->getCodes();
 
         $sql = <<<SQL
-SELECT DISTINCT je.user
-FROM akeneo_batch_job_execution je
-JOIN akeneo_batch_job_instance ji ON je.job_instance_id = ji.id
-WHERE ji.code NOT IN (:not_visible_jobs_codes)
-ORDER BY je.user
-LIMIT :offset, :limit
-SQL;
+            SELECT DISTINCT job_execution.user
+            FROM akeneo_batch_job_execution job_execution
+            JOIN akeneo_batch_job_instance job_instance ON job_execution.job_instance_id = job_instance.id
+            WHERE job_instance.code NOT IN (:not_visible_jobs_codes)
```

Comment:
```suggestion
            WHERE job_execution.is_visible = 1
```
No need to join now that the NotVisibleJobsRegistry has been removed (sorry 🤗 )